macros/streamline/streamline_udfs.sql

```diff
@@ -30,3 +30,13 @@
         'https://smro9shis5.execute-api.us-east-1.amazonaws.com/dev/udf_api'
     {%- endif %};
 {% endmacro %}
+
+{% macro create_udf_hex_encode_function() %}
+    CREATE EXTERNAL FUNCTION IF NOT EXISTS streamline.udf_hex_encode_function(
```

Comment:
how do we update these udfs? this should probably be create or replace without the if not exists.   See the ethtereum-models repo as an example. The udfs only get updated when the variable   [UPDATE_UDFS_AND_SPS](https://github.com/FlipsideCrypto/ethereum-models/blob/7e02c3f2e800a14b32b7db033dbc11dcc8879357/macros/create_udfs.sql#L2) is set